starter-code/src/App.js

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from "react";
+import "./App.css";
+
+class App extends Component {
+  render() {
+    return (
+      <div>
+        <div className="App container">
+          <div className="nav">
+            <nav>
+              <img src="/images/ironhack-logo.svg" />
+              <img src="/images/menu-top.svg" />
+            </nav>
+          </div>
+          <div className="hero">
+            <div className="wrapper">
+              <h1>Say hello to ReactJS</h1>
+              <p className="herop">
+                You will learn a frontend <br />
+                <br />
```

Comment:
adding break tags isn't good practice - it's better to style it by altering the width and padding etc